configure.ac

```diff
@@ -131,6 +131,10 @@ AM_PROG_VALAC([0.20])
 AC_PROG_INSTALL
 AC_PROG_MAKE_SET
 
+m4_ifdef([AX_PROG_CC_FOR_BUILD], [
+AX_PROG_CC_FOR_BUILD
+], [AC_FATAL([AX_PROG_CC_FOR_BUILD not found, you will need to install autoconf-archive])])
```

Comment:
This makes users to install autoconf-archive forcibly. Do you have any ideas to put a warning only that the flag might be important for cross-compile instead of the fatal error? I guess `$(CC_FOR_BUILD)` could be applied to CC in case `$(CC_FOR_BUILD)` is not NULL.

Reply:
The forces the user to install autoconf-archive when doing `autoreconf`, if they are just doing `./configure` without autoconf-archive, it shouldn't display an error (and autoconf-archive shouldn't be necessary after generating `./configure`). Is that fine?